src/views/RoomSelector.vue

```diff
@@ -152,10 +172,18 @@ export default {
 	},
 	methods: {
 		fetchRooms() {
-			axios.get(generateOcsUrl('/apps/spreed/api/v4/room')).then((response) => {
-				this.rooms = response.data.ocs.data.sort(this.sortConversations)
-				this.loading = false
-			})
+			if (this.endPointApi === 'listed-room') {
+				searchListedConversations({ searchText: '' }, {}).then((response) => {
+					this.rooms = response.data.ocs.data.sort(this.sortConversations)
+					this.loading = false
+				})
+
+			} else {
+				axios.get(generateOcsUrl('/apps/spreed/api/v4/room')).then((response) => {
```

Comment:
```suggestion
				fetchConversations({}).then((response) => {
```
And i think, we could refactor it a bit, to avoid duplicates:

```js
const response = this.endPointApi === 'listed-room'
  ? await searchListedConversations({ searchText: '' }, {})
  : await fetchConversations({})

this.rooms = response.data.ocs.data.sort(this.sortConversations)
this.loading = false
```